pvlib/iotools/solcast.py

```diff
@@ -0,0 +1,280 @@
+""" Functions to access data from the Solcast API.
+"""
+import requests
+import pandas as pd
+
+BASE_URL = "https://api.solcast.com.au/data"
+
+
+# TODO: https://pvlib-python.readthedocs.io/en/stable/user_guide/variables_style_rules.html
+PARAMETERS_MAPPING = dict(
+
+)
+
+
+def format_df(df):

```

Comment:
Are type hints common in pvlib? Personally love them but best to just match whatever is most common in the repo

Reply:
nope. I also tend to use them but the style doesn't have them